diff --git a/node_modules/react-native-config/android/dotenv.gradle b/node_modules/react-native-config/android/dotenv.gradle
index 7a2df3b..8cb3a67 100644
--- a/node_modules/react-native-config/android/dotenv.gradle
+++ b/node_modules/react-native-config/android/dotenv.gradle
@@ -79,21 +79,21 @@ android {
 tasks.whenTaskAdded { task ->
     if (project.hasProperty("envConfigFiles")) {
         project.envConfigFiles.each { envConfigName, envConfigFile ->
-            if (task.name.toLowerCase() == "generate"+envConfigName+"buildconfig") {
-                task.doFirst() {
-                    android.applicationVariants.all { variant ->
-                        def variantConfigString = variant.getVariantData().getVariantConfiguration().getFullName()
-                        if (envConfigName.contains(variantConfigString.toLowerCase())) {
-                            loadDotEnv(envConfigName)
-                            project.env.each { k, v ->
-                                def escaped = v.replaceAll("%","\\\\u0025")
-                                variant.buildConfigField "String", k, "\"$v\""
-                                variant.resValue "string", k, "\"$escaped\""
-                            }
-                        }
-                    }
-                }
-            }
+            // if (task.name.toLowerCase() == "generate"+envConfigName+"buildconfig") {
+            //     task.doFirst() {
+            //         android.applicationVariants.all { variant ->
+            //             def variantConfigString = variant.getVariantData().getVariantConfiguration().getFullName()
+            //             if (envConfigName.contains(variantConfigString.toLowerCase())) {
+            //                 loadDotEnv(envConfigName)
+            //                 project.env.each { k, v ->
+            //                     def escaped = v.replaceAll("%","\\\\u0025")
+            //                     variant.buildConfigField "String", k, "\"$v\""
+            //                     variant.resValue "string", k, "\"$escaped\""
+            //                 }
+            //             }
+            //         }
+            //     }
+            // }
         }
     }
 }
